libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -136,8 +136,12 @@ namespace eosio::chain {
     * All methods assert until open() is closed.
     */
    class fork_database {
+      static constexpr uint32_t magic_number = 0x30510FDB;
+
+      enum class in_use_t : uint32_t { legacy, savanna, both };
```

Comment:
maybe `legacy_only`, `savanna_only`, `both`?

Reply:
I don't think it is any clearer, just more verbose.